src/utils/daos.c

```diff
@@ -1071,8 +1071,63 @@ print_oclass_names_list(FILE *stream)
 	free(str);
 }
 
+#define FIRST_LEVEL_HELP() \
+do { \
+	fprintf(stream, \
+	"usage: daos RESOURCE COMMAND [OPTIONS]\n" \
+	"resources:\n" \
+	"	  pool             pool\n" \
+	"	  container (cont) container\n" \
+	"	  object (obj)     object\n" \
+	"	  version          print command version\n" \
+	"	  help             print this message and exit\n"); \
+	fprintf(stream, "\n"); \
+	fprintf(stream, "use 'daos help RESOURCE' for resource specifics\n");
+} while (0)
+
+#define ALL_CONT_CMDS_HELP() \
```

Comment:
(style)  macros should not use a trailing semicolon

Reply:
should have been resolved with next commit 7c41b2cd051243cf4560a225393628316c0a9901